### ISSUE TITLE
Spike: Propose config recommendation for AWS WAF

### ISSUE THREAD

**Issue by Ben-Hodgkiss** [digital-land/technical-documentation#64]
**Overview**

Investigate where AWS WAF (Web Application Firewall) should be employed across the Planning Data Service and provide a configuration recommendation.

Recommendation should detail which rules should be implemented for each WAF ACL (Access Control List).

A price estimate for the recommended configuration should also be included in the proposal.

**Tech Approach**

A bullet pointed list with details on how this could be technically worked.

- Research good practice within MHCLG and within the industry for AWS WAF
- Provide AWS infrastructure diagram with recommended WAFs
- Use AWS cost calculator for price estimate
- Document recommendation (file location TBA)

**Acceptance Criteria/Tests**

- Documented proposal which includes
  - overview on good practice researched
  - recommended configuration (ACL rules etc)
  - price estimate

‌

**Ticket Management - DELETE** this section once completed

- Complete all relevant tags - make sure Infrastructure is tagged so it is picked up by our filters!
- Complete the time estimate field
- Make sure you have a PR link in the comments.
- If relevant, link to the relevant OKR as an attachment.
- Link to any tickets in other boards that are dependent on it - i.e. Data Management or Providers tickets.

**Comment by cpcundill** [comment]
AWS Cost Calculator estimate here:

https://calculator.aws/#/estimate?id=276e8e7b782d2934758c57d4383865cfecb429d2

**Comment by cpcundill** [comment]
Added discussion here:

https://github.com/digital-land/digital-land-infrastructure/discussions/241

**Comment by cpcundill** [comment]
Agreed to move forward at AWS review meeting this morning.  Implementation ticket created here:

https://github.com/digital-land/digital-land-infrastructure/issues/244
